nxontology_ml/features.py

```diff
@@ -50,3 +50,35 @@ def __init__(self, nxo: NXOntology[str] | None = None):
     def transform(self, X: list[str], copy: bool | None = None) -> NodeFeatures:
         # X are nodes
         return NodeFeatures.from_nodes(nodes=[self._nxo.node_info(n) for n in X])
+
+
+class SubsetsFeatures(DataFrameFnTransformer):
```

Comment:
Would be nice to include a link to https://github.com/related-sciences/nxontology-ml/issues/14 somewhere in this class, possibly via a docstring